### DISCUSSION TITLE
pine needle

### DISCUSSION THREAD

**Discussion by SaharRasoulpour** [firemodels/fds#13363] [Q&A]
Following the discussion of #13130,
In my case, after a specific time, when the bed of pine needles reaches 600°C (which occurs at 45seconds in the middle of the simulation), I observe a "breathing" effect in the fire—where the temperature fluctuates as if the fire is inhaling and exhaling. How can I achieve a smoother, more gradual reduction in temperature within the bed? Which parameters and factors are essential to address this?

**Comment by rmcdermo** [comment]
First thing I would do is try the [latest nightly build](https://github.com/firemodels/test_bundles/releases/tag/FDS_TEST).  We have recently implemented a new scheme for handling oxygen mass transfer in oxidative pyrolysis which may smooth out the effect you are seeing.

**Reply by SaharRasoulpour** [reply]
I will take it into consideration, but when I ran the USFS_Corsica sample case with my adjusted parameters and configuration, the aforementioned domain showed no fluctuations. However, with these adjustments, I encountered one problem: the peak temperature was lower than expected.
